### ISSUE TITLE
Someone help me with this bug

### ISSUE THREAD

**Issue by ghost** [haskellcamargo/skype-unofficial-client#13]
josue@RegataOS-desktop:~> /opt/skype_unofficial_client/skype
[5785:1019/003037:ERROR:browser_main_loop.cc(170)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
[5814:1019/003039:WARNING:nw_render_view_observer.cc(106)] Failed to load js script file: /opt/skype_unofficial_client/index.js
[5814:1019/003041:WARNING:nw_render_view_observer.cc(106)] Failed to load js script file: /opt/skype_unofficial_client/index.js
[5814:1019/003045:WARNING:nw_render_view_observer.cc(106)] Failed to load js script file: /opt/skype_unofficial_client/index.js
[5814:1019/003046:WARNING:nw_render_view_observer.cc(106)] Failed to load js script file: /opt/skype_unofficial_client/index.js

**Comment by ryanwarsaw** [comment]
@dantix is this related to your commit?

**Comment by roman-holovin** [comment]
@ryanw-se yes, it is. Looking into it.
@regataos can you provide info about what distro are you using and what installation method you are using?

**Update**
I installed it from Arch AUR (it basically cloning this repo and calling make) and it is working fine on 3 different computers, so waiting for more info to find whats wrong with it.

**Update 2** 
Installed using instruction provided in README just in case. Working fine also.

**Comment by ghost** [comment]
I'm using openSUSE. Another problem that also realize is that Skype is reconnecting direct.
I installed with the instructions that are on the homepage of the project, basically:

git clone https://github.com/haskellcamargo/skype-unofficial-client.git
 sudo make linux_x64
 /opt/skype_unofficial_client/skype

The log mention "index.js" file, which was apparently missing after installation. I added the index.js file in the application installation directory, but Skype still reconnecting.

**Comment by ghost** [comment]
RegataOS-desktop:/home/josue/skype-unofficial-client # /opt/skype_unofficial_client/skype
[5841:1019/155912:ERROR:browser_main_loop.cc(170)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
[5841:1019/155914:INFO:CONSOLE(2)] "Uncaught ReferenceError: require is not defined", source:  (2)
[5841:1019/155917:INFO:CONSOLE(2)] "Uncaught ReferenceError: require is not defined", source:  (2)
[5841:1019/155920:INFO:CONSOLE(2)] "Uncaught ReferenceError: require is not defined", source:  (2)
[5841:1019/155923:INFO:CONSOLE(2)] "Uncaught ReferenceError: require is not defined", source:  (2)

As we reported before, Skype is reconnecting several times. Follows a link to a sample image: https://drive.google.com/file/d/0B6R1x3fHyLx2bFBnOE94Y09weHM/view?usp=sharing

**Comment by roman-holovin** [comment]
@regataos I've created pull request, that reverts my commit since it is not really needed (and is not working since I did not provide appropriate change to makefile, that should copy index.js to /opt directory).

Missing file index.js, as well "require is not defined", are not critical errors, Skype continues to work for me  even after these errors.

As for your error, it seems to be unrelated to all of this mess.

Is web version of skype working for you in regular browser? (http://web.skype.com)
